arviz/data/io_cmdstan.py

```diff
@@ -212,6 +212,8 @@ def sample_stats_to_xarray(self):
                 key_, *end = key.split(".")
                 name = re.sub("__$", "", key_)
                 name = "diverging" if name == "divergent" else name
+                name = "n_steps" if name == "n_leapfrog" else name
+                name = "tree_depth" if name == "treedepth" else name
```

Comment:
I think that now that multiple variables are renamed, using a dict as mapping like in `io_numpyro` will be more clear